odl/space/entry_points.py

```diff
@@ -46,12 +46,17 @@
 standard_library.install_aliases()
 
 from pkg_resources import iter_entry_points
+
 from odl.space.npy_ntuples import NumpyNtuples, NumpyFn
+from odl.space.npy_tensors import NumpyTensorSet, NumpyTensorSpace
 
 __all__ = ('NTUPLES_IMPLS', 'FN_IMPLS')
 
+
 NTUPLES_IMPLS = {'numpy': NumpyNtuples}
+TENSOR_SET_IMPLS = {'numpy': NumpyTensorSet}
 FN_IMPLS = {'numpy': NumpyFn}
+TENSOR_SPACE_IMPLS = {'numpy': NumpyTensorSpace}
```

Comment:
This doesn't do anything right now. Probably it can vanish from this PR but later on there needs to be a transition here, too.